### PR TITLE
Update project status to sunset

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
## Which problem is this PR solving?
Updates the project status to sunset.

## Short description of the changes
- Updates OSSMETADATA to status to sunset
- Updates OSS badge to pink and add link to issue

### CODE REVIEW

**Review by JamieDanielson — COMMENTED** [review]
I think we want to hold off a tiny bit longer until the [updates](https://github.com/honeycombio/docs/pull/1071) get merged into public docs for python sampling with OTel...

**Comment by MikeGoldsmith** [comment]
Adding `blocked` until https://github.com/honeycombio/docs/pull/1071 is merged.

**Review by JamieDanielson — APPROVED** [review]
Approving so it's ready to go once the public docs changes are merged in.

**Pull request merged**
